test/fgpu/test_postproc.py

```diff
@@ -64,46 +65,51 @@ def postproc_host_pol(
     phase = np.exp(m2jpi * fine_delay[:, np.newaxis] * channel_idx / (2 * channels) + 1j * fringe_phase[:, np.newaxis])
     assert phase.dtype == np.complex64
     # Apply delay, phase and gain
+    corrected: np.ndarray
```

Comment:
```bash
test/fgpu/test_postproc.py:70: error: Incompatible types in assignment
(expression has type "ndarray[Any, dtype[floating[_32Bit]]]", 
variable has type "ndarray[Any, dtype[bool_]]")  [assignment]

test/fgpu/test_postproc.py:80: error: Incompatible types in assignment 
(expression has type "ndarray[Any, dtype[complexfloating[_32Bit, _32Bit]]]", 
variable has type "ndarray[Any, dtype[bool_]]")  [assignment]
```

I see.

Reply:
Yeah I should add a comment to explain that this is to work around wonky type inference. I haven't figured out there the `bool_` is coming from.